config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml

```diff
@@ -19,7 +20,9 @@ spec:
           transition: GreetInEnglish
         - condition: "${ .language == \"Spanish\" }"
           transition: GreetInSpanish
-      defaultCondition: GreetInEnglish
+      defaultCondition:
+        transition:
+          nextState: GreetInEnglish
```

Comment:
Same thing here: https://github.com/serverlessworkflow/specification/blob/0.8.x/schema/workflow.json#L286-L323

It should be a `StringOrObject` struct.

Reply:
> @ricardozanini @davidesalerno It is ready for review, I still need to test it on a cluster as well.

most of the issues seems to be cause due wrong field descriptions, e.g.: 
https://github.com/serverlessworkflow/sdk-go/blob/main/model/action_data_filter.go#L25

Reply:
will address it in the sdk.
https://github.com/serverlessworkflow/sdk-go/issues/157

will address this one after I finish to review the e2e openApi faliures.

Reply:
resolved.